### PR TITLE
Bump svgo to 0.7.0

### PR DESCRIPTION
**Pull request by gtrufitt**
https://github.com/svg/svgo/releases/tag/0.7.0

Thanks for grunt-svgmin ❤️

### CODE REVIEW

**Comment by kevva** [comment]
From their [changelog](https://github.com/svg/svgo/blob/master/CHANGELOG.md#---070--25082016):

> Required Node.js version has increased to 0.12

Could you remove `0.10` from the `.travis.yml` and change the minimum version in `package.json`?

**Comment by gtrufitt** [comment]
Thanks @kevva - updated and added node 6

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
+  - '6'
   - '5'
```
Comment (gtrufitt) [inline .travis.yml]: Done 👍  What's the convention here - minimum, stable and latest?

**Comment by kevva** [comment]
Thanks 👍

**Pull request merged**